### DISCUSSION TITLE
Hi, I would like to know why the majority of time series segmentation studies are unsupervised.

### DISCUSSION THREAD

**Discussion by 98brucelee** [lzz19980125/awesome-time-series-segmentation-papers#4] [Q&A] [answered]
Hi, I would like to know why the majority of time series segmentation studies are unsupervised.

**Comment by lzz19980125** [comment, marked as answer]
Thank you for your interest. As mentioned in the README: Manually annotating segment points (change points) in large time series datasets is extremely labor-intensive and somewhat subjective. Therefore, the field of time series segmentation lacks large public datasets with ground truth, making it difficult for supervised methods to find sources of training data. Unsupervised time series segmentation acts to some extent as an automatic annotator of segmentation points, making it easier to implement.
